README.md

```diff
@@ -60,6 +60,10 @@ browserstack.username=<browserstack_username>
 browserstack.password=<browserstack_password>
 data.location=develop
 api.keys=<api_key>
+app.token.url=https://login.microsoftonline.com/<azure_app_id>/oauth2/v2.0/token
+app.client.id=<app_client_id>
```

Comment:
Nice work so this will now allow us to get the tokens for our different apps registered under our tenant?
Question for us, how do we share these values when we onboard the testers? is that on a confluence page and do we want to link it back here?

Reply:
This can be shared once the testers are onboarded, access can be granted accordingly, hence it won't be required to store in anywhere. I can guide through this process.